templates/gsg-trigger.patch

```diff
@@ -1,6 +1,6 @@
 diff -Naur create-daml-app/daml/ChatBot.daml gsg-trigger/daml/ChatBot.daml
 --- create-daml-app/daml/ChatBot.daml	1970-01-01 01:00:00.000000000 +0100
-+++ gsg-trigger/daml/ChatBot.daml	2021-08-16 13:05:51.000000000 +0200
++++ gsg-trigger/daml/ChatBot.daml	2021-08-18 13:15:25.000000000 +0200
```

Comment:
instead of inlining this into this patch could we first apply `messaging.patch` and then this one to avoid the two getting out of sync?

Reply:
I wish we could, but `messaging.patch` applies on a realized version of the template, not on the template itself. Fixing that seemed like a bit more effort than I thought reasonable to put into this at this point.

Reply:
makes sense, thanks for the explanation